.gitignore

```diff
@@ -1,4 +1,2 @@
-.idea/
```

Comment:
`/.idea/` byl odstraněn náhodně? Mohli bychom ho vrátit?

Reply:
@BadKoldrysh `.gitignore` (ve veřejném projektu) by se měl týkat vždy jen a pouze projektu samotného. To `.idea` nesplňuje, protože se jedná o interní individuální potřebu každého vývojáře, který by vyřazení takového adresáře měl řešit na úrovni nastavení vlastního OS.

Toho jde docílit použitím globálního `.gitignore` v OS. Zpravidla to bývá soubor `~/.gitignore` popř. `~/.gitignore_global`, co z toho je na tvém OS použito, se dozvíš zavoláním `git config core.excludesfile`.

Reply:
Aha, rozumím, děkuju